### PR TITLE
Added BOA Token (BOSAGORA)

### PR DESCRIPTION
**Pull request by TrustHenry**
Added BOA Token (BOSAGORA)
https://www.bosagora.io

### CODE REVIEW

File: src/tokens/eth/0x746DdA2ea243400D5a63e0700F190aB79f06489e.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "symbol": "BOA",
+    "name": "BOSAGORA",
+    "type": "ERC20",
+    "address": "0x746DdA2ea243400D5a63e0700F190aB79f06489e",
+    "ens_address": "",
+    "decimals": 7,
+    "website": "https://www.bosagora.io",
```
Comment (Geod24) [inline src/tokens/eth/0x746DdA2ea243400D5a63e0700F190aB79f06489e.json]:
```suggestion
    "website": "https://bosagora.io",
```

File: src/tokens/eth/0x746DdA2ea243400D5a63e0700F190aB79f06489e.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      "forum": "",
+      "github": "https://github.com/bpfkorea/bosagora-erc20",
+      "gitter": "",
+      "instagram": "",
+      "linkedin": "",
+      "reddit": "",
+      "slack": "",
+      "telegram": "https://t.me/bpf_eng",
+      "twitter": "",
+      "youtube": "https://www.youtube.com/channel/UCjcTwkskyTmAwHpqv9Oynig?view_as=subscriber"
```
Comment (Geod24) [inline src/tokens/eth/0x746DdA2ea243400D5a63e0700F190aB79f06489e.json]:
```suggestion
      "youtube": "https://www.youtube.com/channel/UCjcTwkskyTmAwHpqv9Oynig"
```

File: src/tokens/eth/0x746DdA2ea243400D5a63e0700F190aB79f06489e.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+    "symbol": "BOA",
+    "name": "BOSAGORA",
+    "type": "ERC20",
+    "address": "0x746DdA2ea243400D5a63e0700F190aB79f06489e",
+    "ens_address": "",
+    "decimals": 7,
+    "website": "https://www.bosagora.io",
+    "logo": { "src": "https://bosagora.io/images/boaLogo.png", "width": "256", "height": "256", "ipfs_hash": "" },
+    "support": { "email": "tokennet@bpfkorea.org", "url": "https://www.bosagora.io" },
```
Comment (Geod24) [inline src/tokens/eth/0x746DdA2ea243400D5a63e0700F190aB79f06489e.json]:
```suggestion
    "support": { "email": "tokennet@bpfkorea.org", "url": "https://bosagora.io" },
```

**Comment by AndrejMitrovic** [comment]
I think you should squash the two commits?

**Comment by TrustHenry** [comment]
merge Plz..

**Pull request merged**